cli-toolchain/quickstart.md

```diff
@@ -397,6 +398,7 @@ $ amplify add api
 ? Choose the function template that you want to use:
   CRUD function for Amazon DynamoDB table (Integration with Amazon API Gateway and Amazon DynamoDB)
 ❯ Serverless express function (Integration with Amazon API Gateway)
+  Lambda trigger
```

Comment:
Is this just showing in the `amplify add api` section because that kicks the same `add function` flow? Seems a bit confusing to show this first.

Reply:
Yeah, the `amplify add api` kicks off the add function flow - that's why it shows up out here.